doc/sphinx/docs/python/tutorials/coding_udp_multi_objective.rst

```diff
@@ -0,0 +1,71 @@
+.. _py_tutorial_coding_udp_multi_objective:
+
+Coding a User Defined Problem with constraints multiple objectives
```

Comment:
with constraints multiple objectives should be "with multiple objectives" ?

Reply:
You can skip the CI for these type of typo fixes by embedding "[SKIP CI]" in your commit message (i think in front)

Reply:
Done! Weird that I did not notice the headline...

Have you checked the rendering of the mathematical description? I did not build the docs locally...